resource_customizations/kafka.strimzi.io/KafkaConnect/health.lua

```diff
@@ -0,0 +1,21 @@
+hs = {}
+if obj.status ~= nil then
+  if obj.status.conditions ~= nil then
+    for i, condition in ipairs(obj.status.conditions) do
+      if condition.type == "Ready" and condition.status == "False" then
```

Comment:
Hm, I am wondering whether the state should be really set to `Degraded` here. Could it be that that the real state is still progressing, when `Ready` condition is `False`? Is there another condition to check whether the operator has failed where we can set `Degraded` status?

Reply:
I'll have a dig into what the CRD exposes! Thanks for the comment.

Reply:
I'm looking into this, having a chat with the Strimzi guys.

Reply:
@jannfis Updated!

Reply:
Great, thank you @chris-vest! One last request: Can you please rebase your PR with most recent master, and then I'm going to merge it if all tests pass.

Reply:
Sure!

Reply:
Should be rebased!